internal/rcm/api.go

```diff
@@ -42,7 +79,47 @@ func New(logger *log.Logger, store *store.Store, rpmMetadata rpmmd.RPMMD) *API {
 	api.router.MethodNotAllowed = http.HandlerFunc(methodNotAllowedHandler)
 	api.router.NotFound = http.HandlerFunc(notFoundHandler)
 
+	// swagger:route POST /v1/compose submit
+	//
+	// Submit requests for composes.
+	//
+	//     Consumes:
+	//     - application/json
+	//
+	//     Produces:
+	//     - application/json
+	//
+	//     Schemes: http
+	//
+	//     Responses:
+	//       default: errorReason
+	//       200: submitReply
 	api.router.POST("/v1/compose", api.submit)
+
+	// swagger:operation GET /v1/compose/{uuid} status
+	//
+	// Check current status of a running compose.
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: uuid
+	//   in: path
+	//   description: uuid to display status for
+	//   required: true
+	//   type: string
+	// responses:
+	//   '200':
+	//     description: status of a running compose
+	//     schema:
+	//       type: object
+	//       items:
+	//         "$ref": "#/definitions/statusReply"
+	//   default:
+	//     description: unexpected error
+	//     schema:
+	//       "$ref": "#/definitions/errorReason"
```

Comment:
Are we sure we want to put this comment here rather than in front of the `status` function definition?

Reply:
It does not matter, it is just a personal preference.